includes/class-sensei-lesson.php

```diff
@@ -4253,8 +4253,9 @@ public static function footer_quiz_call_to_action( $lesson_id = 0, $user_id = 0
 
 		if ( intval( $lesson_prerequisite ) > 0 ) {
 
-			// If the user hasn't completed the prereq then hide the current actions
-			$show_actions = Sensei_Utils::user_completed_lesson( $lesson_prerequisite, $user_id );
+			// If the user hasn't completed the prereq then hide the current actions.
+			// (If the user is either the lesson creator or admin, show actions)
+			$show_actions = Sensei_Utils::user_completed_lesson( $lesson_prerequisite, $user_id ) || (int) get_post_field( 'post_author', $lesson_id ) === $user_id || current_user_can( 'create_users' );
```

Comment:
This functionally works! Maybe it'd be nice to extract these out into some helper functions? Like `Sensei_Utils::is_lesson_author()`, `Sensei_Utils::is_administrator()`, etc. Also I think `current_user_can( 'manage_options' )` is typically what's used to check for an administrator role.

And a code formatting thing, it's good to wrap long conditionals to new lines:
`if (
    $condition1
    || $condition2
    || $condition3
) {
    // Code here
} `

Reply:
Hey Dan! Great, I'll work on your suggestions ASAP... Thanks for the feedback!

Reply:
Hey Dan! I just finished working on your suggestions... Let me know your thoughts and if there's anything else I could do to help!